packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/iam/iam.html

```diff
@@ -0,0 +1,62 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row" data-ng-if="!$ctrl.loading">
+    <div class="col-sm-9">
+        <dl>
+            <dd data-translate="dedicatedCloud_USER_iam_description"></dd>
+            <dd>
+                <small
+                    data-translate="dedicatedCloud_USER_iam_usage_warning"
+                ></small>
+            </dd>
+            <dt data-translate="dedicatedCloud_USER_iam_status"></dt>
+            <dd>
+                <ovh-manager-pcc-dashboard-component-activation-status
+                    class="d-block mt-1"
+                    data-status-name=":: $ctrl.iamStatus.state"
+                ></ovh-manager-pcc-dashboard-component-activation-status>
+            </dd>
+        </dl>
+    </div>
+    <div class="col-sm-3">
+        <button
+            type="button"
+            class="mb-3 btn btn-block btn-default"
+            data-ng-if="$ctrl.iamEnabled"
+            data-ng-click="$ctrl.onGoToToggleIamClick('disable')"
+            data-ng-disabled="!$ctrl.iamTogglable"
+            data-oui-tooltip="{{ $ctrl.iamTogglableError ? ('dedicatedCloud_USER_iam_api_error_returned' | translate:{ message: $ctrl.iamTogglableError }) : '' }}"
+            data-oui-tooltip-placement="bottom"
+        >
+            <span data-translate="dedicatedCloud_USER_iam_disable"></span>
+        </button>
+        <button
+            type="button"
+            class="mb-3 btn btn-block btn-default"
+            data-ng-if="$ctrl.iamDisabled"
+            data-ng-click="$ctrl.onGoToToggleIamClick('enable')"
+            data-ng-disabled="!$ctrl.iamTogglable"
+            data-oui-tooltip="{{ $ctrl.iamTogglableError ? ('dedicatedCloud_USER_iam_api_error_returned' | translate:{ message: $ctrl.iamTogglableError }) : '' }}"
+            data-oui-tooltip-placement="bottom"
+        >
+            <span data-translate="dedicatedCloud_USER_iam_enable"></span>
+        </button>
```

Comment:
**suggestion:** These 2 buttons can be combined
```suggestion
        <button
            type="button"
            class="mb-3 btn btn-block btn-default"
            data-ng-click="$ctrl.onGoToToggleIamClick($ctrl.iamEnabled ? 'disable' : 'enable')"
            data-ng-disabled="!$ctrl.iamTogglable"
            data-oui-tooltip="{{ $ctrl.iamTogglableError ? ('dedicatedCloud_USER_iam_api_error_returned' | translate:{ message: $ctrl.iamTogglableError }) : '' }}"
            data-oui-tooltip-placement="bottom"
        >
            <span data-ng-if="$ctrl.iamEnabled" data-translate="dedicatedCloud_USER_iam_disable"></span>
            <span data-ng-if="!$ctrl.iamEnabled" data-translate="dedicatedCloud_USER_iam_enable"></span>
        </button>
        <button
            type="button"
            class="mb-3 btn btn-block btn-default"
            data-ng-if="$ctrl.iamDisabled"
            data-ng-click="$ctrl.onGoToToggleIamClick('enable')"
            data-ng-disabled="!$ctrl.iamTogglable"
            data-oui-tooltip="{{ $ctrl.iamTogglableError ? ('dedicatedCloud_USER_iam_api_error_returned' | translate:{ message: $ctrl.iamTogglableError }) : '' }}"
            data-oui-tooltip-placement="bottom"
        >
            <span data-translate="dedicatedCloud_USER_iam_enable"></span>
        </button>
```

Reply:
I had to change slightly what you suggested because the buttons are displayed only when option is disabled or enabled, not the other states.